scripts/update-all.sh

```diff
@@ -0,0 +1,7 @@
+#! /bin/sh
+
+NEW_VERSION=$(npm show prisma@$1 version)
+echo "$NEW_VERSION" > .github/prisma-version.txt
```

Comment:
Isn't this called with an explicit version string in $1, and doing these two things will jsut return the same again?

Reply:
Yes, it's a small tweak for running `update-all` (eg `update-all dev`) locally while `update-ci` is meant just for ci.

Reply:
(can this be marked as resolved?)

Reply:
Added a comment